File: src/main/java/no/fintlabs/access/control/repository/ResourceAccessEntityRepository.java
package no.fintlabs.access.control.repository;

import jakarta.validation.constraints.NotNull;
import no.fintlabs.access.control.model.entity.ClientEntity;
import no.fintlabs.access.control.model.entity.PackageAccessEntity;
import no.fintlabs.access.control.model.entity.ResourceAccessEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Collection;

public interface ResourceAccessEntityRepository extends JpaRepository<ResourceAccessEntity, Integer> {
    Collection<ResourceAccessEntity> findByPackageId(Integer id);
}

File: src/main/java/no/fintlabs/access/control/repository/ResourceAccessEntityRepository.kt
package no.fintlabs.access.control.repository

import no.fintlabs.access.control.model.entity.ResourceAccessEntity
import org.springframework.data.jpa.repository.JpaRepository

interface ResourceAccessEntityRepository : JpaRepository<ResourceAccessEntity, Int> {
    fun findByPackageId(id: Int): Collection<ResourceAccessEntity>
}
